storage.go

```diff
@@ -168,6 +168,7 @@ func (s *Storage) Delete(ctx context.Context, k microstorage.K) error {
 	var body []byte
 	{
 		v := storageConfigJSONPatch{}
+		v.Spec.Storage.Data = map[string]*string{}
```

Comment:
For my interest - why do we need to initialise this map now, but didn't need to before?

Reply:
Because the test showed that it panics when it is not initialized. Because the zero value of a map is `nil` and try to access something on nil panics.